HousingRepairsOnlineApi/UseCases/IRetrieveAvailableAppointmentsUseCase.cs

```diff
@@ -1,11 +1,13 @@
 ﻿using System.Collections.Generic;
 using System.Threading.Tasks;
 using HACT.Dtos;
```

Comment:
This `using` is likely (and should be) redundant now that the return type no longer comes from `HACT.Dtos` - if so, please delete.